woocommerce/wc-admin
# Inbox: Remove Store Setup Alert

Fixes #5460 

This PR removes the "Welcome to WooCommerce! Set up your store and start selling" alert, as it has be superseded functionality-wise by the Activity Panel Store Setup link.

![](https://user-images.githubusercontent.com/22080/96895999-e3ab0d80-1441-11eb-945d-a2cec795feee.png)

### Detailed test instructions:

- Run `main` branch

- Update DB to ensure that the store setup alert is visible

```
UPDATE wp_wc_admin_notes
SET status = 'unactioned'
WHERE name = 'wc-admin-onboarding-profiler-reminder'
```

- Verify store setup alert is visible

- Checkout and run branch

- With WP-CLI, update the `woocommerce_admin_version` option so that the WooCommerce Admin plugin thinks it needs to upgrade and runs the upgrade logic (which will delete the obsolete notes)

```
wp option update woocommerce_admin_version 0.1
```

- Verify store setup alert is no longer visible

### Changelog Note:



Tweak: Remove store setup alert.
